CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# minimum cmake version supported
```

Comment:
What's this file?

Reply:
You mentioned that the "consolidate" step is done in C. Let's make sure to move that to Go next. Ideally index creation will ALL occur in Go. It's a nice way to transition I think.

Reply:
To clarify: we don't have to do this in this PR. Just a note for the project.

Reply:
I agree. I will remove this file. I need it as part of my build so there's only a single step for the user (the user doesn't have to build go and then build c++ -- they just build the project). It doesn't have to be in this repo since it's pure go.